src/DotNet.Testcontainers.Tests/DotNet.Testcontainers.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="CouchbaseNetClient" Version="3.0.1" />
```

Comment:
Latest version `3.0.3`.

Reply:
I am not sure the latest version is stable especially about N1QL calls. But latest version can cover to test Couchbase Testcontainer. I will update.